frontend/turbo.json

```diff
@@ -1,29 +1,16 @@
 {
     "$schema": "https://turborepo.org/schema.json",
     "globalEnv": [
-        "INTERNET_IDENTITY",
```

Comment:
Shouldn't we now have BUILD_ENV in this list?

Reply:
Turbo only loads env variables from the .env file in the frontend directory, so it isn't aware of BUILD_ENV.
Also if we change the BUILD_ENV in any of the build scripts that would count as an input file change and so would invalidate the cache.